docs/introduction/domain-settings.md

```diff
@@ -0,0 +1,147 @@
+
+- **Multijazyčný atribut** - Hodnota tohoto atributu muže být nastavena ruzně pro každý jazyk.
+Příkladem multijazyčného atributu je název produktu.
+Příkladem nemultijazyčného atributu je cena produktu. 
+
+- **Rozdíl mezi multidoménovým a multijazyčným atributem** - Hodnota multijazyčného atributu bude stejná pro každou doménu se stejným jazykem.
+Např v případě, že je pro jazyk *en* nastaven název produktu *A4tech mouse*, bude tento název stejný pro všechny domény, které mají nastaven jazyk *en*.
+V případě multidoménového atributu je možné nastavovat ruzné hodnoty pro ruzné domény bez ohledu na jazyk
+
+*Pozn: Demodata, které jsou na Shopsys Frameworku k dispozici obsahují data jen v cz a en jazyce*
+
+## Nastavovaní a práce s doménami
+
+### 1. Vytvoření aplikace s jednou doménou
+
+#### 1.1 Konfigurace domény
+Upravte konfiguraci domény v `app/config/domains.yml`.
+V tomto souboru se nastavuje ID domény, identifikátor domény pro doménové záložky v administraci, jazyk domény.
+
+#### 1.2 Konfigurace url adresy
+Nastavte požadovanou url adresu domény v `app/config/domains_urls.yml`.
+
+#### 1.3 Nastavevení projektu jako jednodoménový
+Upravte nastavení parametru `is-multidomain` v `build.xml` na hodnotu `false`.
+Prostřednictvím této změny se upraví chování některých phingových targetu - např při importu demo dat se aplikace neppokusí importovat zbytečně data pro další doménu.
+
+#### 1.4 Nastavení jazyka
+Nastavení jazyka domény proveďte podle instrukcí ze sekce [Nastavení jazyka](#3-nastaveni-jazyka)
+
+#### 1.5 Build
+Spusťte build, např prostřednictvím phingového targetu 
+```
+php phing build-demo-dev
+```
+Po proběhnutí buildu je již vytvořena jednodoménová aplikace
+
+### 2. Přidání nové domény
+
+#### 2.1 Konfigurace domény
+Přidejte konfiguraci nové domény do `app/config/domains.yml`.
+V tomto souboru se nastavuje ID domény, identifikátor domény pro doménové záložky v administraci, jazyk domény.
+
+#### 2.2 Konfigurace url adresy
+Nastavte url adresu nové domény v `app/config/domains_urls.yml`.
+
+*Pozn: V případě, že přidávate novou doménu na platformě Mac, musíte tuto doménu povolit i v interfasu networku, viz [Installation Using Docker for MacOS](https://github.com/shopsys/shopsys/blob/master/docs/installation/installation-using-docker-macos.md#11-enable-second-domain-optional)
+
+#### 2.3 Nastavevení projektu jako multidoménový
+Upravte nastavení parametru `is-multidomain` v `build.xml` na hodnotu `true`.
+Prostřednictvím této změny se upraví chování některých phingových targetu - např při importu demo dat se naimportují data pro další doménu v případě, že jsou k dispozici.
+
+#### 2.4 Nastavení jazyka
+Nastavení jazyka domény proveďte podle instrukcí ze sekce [Nastavení jazyka](#3-nastaveni-jazyka)
+
+#### 2.5 Nastavení multidoménových atributu 
+Pro správne fungování nové domény je nutno nastavit hodnoty multidoménových atributu pro tuto novou doménu.
+Pro nastavení hodnot multidoménových atributu spusťte phingový target
```

Comment:
If you add that `php phing create-domains-data` call to the 3. Locale settings section, I would just slightly modify this sentence - "Pro nastavení hodnot multidoménových atributu spusťte phingový target (pokud jste tak již neučinili při nastavení jazyka v předchozím kroku)"